lua/nord/theme.lua

```diff
@@ -773,7 +773,7 @@ theme.loadPlugins = function()
 		AerialArrayIcon = { fg = nord.nord13_gui },
 		AerialBooleanIcon = { fg = nord.nord9_gui, style = "bold" },
 		AerialClassIcon = { fg = nord.nord9_gui },
-		AerialConstansIcon = { fg = nord.nord13_gui },
```

Comment:
Searching the Aerial repo this seems to be a typo
https://github.com/stevearc/aerial.nvim/blob/d22daba7af04ffa0363e8fbb0e1c06eab79c647d/lua/aerial/highlight.lua#L38